### ISSUE TITLE
Simple main effects not working in RN Anova

### ISSUE THREAD

**Issue by MarcusFV** [jasp-stats/jasp-issues#1182]
Jasp is returning an error when I try to do simple main effect on RM Anova taking Group as simple effect factor.
The error is

This analysis terminated unexpectedly. 
Error in pf(F, df, fullAnovaDf, lower.tail = FALSE): Non-numeric argument to mathematical function
Stack trace
To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports

Thanks.

**Comment by MarcusFV** [comment]
I am running Jasp in Windows 10.
My Jasp version is 0.14.1. Old versions is working fine.
The complete error message is:

This analysis terminated unexpectedly. 
Error in pf(F, df, fullAnovaDf, lower.tail = FALSE): Non-numeric argument to mathematical function
Stack trace
tryCatchOne(expr, names, parentenv, handlers[[1]])
doTryCatch(return(expr), name, parentenv, handler)
withCallingHandlers(expr = analysis(jaspResults = jaspResults, dataset = dataset, options = options), error = .addStackTrace)
analysis(jaspResults = jaspResults, dataset = dataset, options = options)
.rmAnovaSimpleEffects(rmAnovaContainer, dataset, longData, options, ready)
pf(F, df, fullAnovaDf, lower.tail = FALSE)

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports

**Comment by JohnnyDoorn** [comment]
Hi @MarcusFV,

Thanks for your report. Would it be possible for you to share the jasp file or data set, so that I can look into this issue? You can also send it by email to j<dot>b<dot>vandoorn<at>uva.<dot>nl


Kind regards,
Johnny

**Comment by JasperRobinson** [comment]
Me too. 

I'm trying to drill down into a mixed ANOVA interaction. I remember this working on older versions but neither 0.14.0 nor 0.14.1 seem to run. I can't remember which older version last worked. 

I'm on macOS 11.4

**Comment by JasperRobinson** [comment]
Thank you. I can't share the data file with you—it contains confidential data. 

But, I found the same problem using a sample data set from Jasp. This has the same structure as my data set: 1 between subject variable; 1 within-subject variable. 

This was under ANOVA heading and was Andy Field's "Erotic Pictures & Love". Gender was a 2-level between-subject variable and the 3 continuous variables were a within-subject. I ran it as a Repeated Measures ANOVA—which was fine—then went down to the SIMPLE MAIN EFFECTS tab and set Simple Effect Factor as Gender and the Moderator factor as the other, within-subject, variable. 

Yesterday this showed the same error as the OP's. Today it is fine. :-/

My actual data set fails today too. If I set it with the within-subject variable as the simple effect factor, instead of the between subject factor, it does run. The data set has a lot of zeros in the first few levels of the within-subject factor. Maybe this is causing the error; but I don't understand why the Andy Field data should go wrong too.
